populus/compilation/backends/solc_auto.py

```diff
@@ -13,40 +14,32 @@
     is_string,
 )
 
-from .base import (
-    BaseCompilerBackend,
-)
 from .solc_combined_json import (
     SolcCombinedJSONBackend,
 )
+
 from .solc_standard_json import (
     SolcStandardJSONBackend,
 )
 
 
-def get_solc_backend_class_for_version(solc_version):
-    if is_string(solc_version):
-        solc_version = Version(solc_version)
+def get_compiler_backend_class_for_version(compiler_version, global_config=None):
```

Comment:
The rename from` solc` to `compiler` doesn't feel right here, because this function is all solc specific.  When/if we introduce support for viper, we'll want to have this distinction clearly in place.